tensorflow_io/lmdb/python/ops/lmdb_ops.py

```diff
@@ -41,28 +41,33 @@ def __init__(self, filenames):
       print(key, value)
     ```
     Args:
-      filenames: A `tf.string` tensor containing one or more filenames.
+      filename: A `tf.string` tensor containing one or more filenames.
```

Comment:
Is there a particular reason for this change?

Reply:
The input is slightly different than before, in that now it is a 1D or 2D tensor (not a list of tensors). Think it makes sense to remove s as we consider tensor as singular in Tensorflow overall.

Reply:
This would break backward compatibility though. Should we add a deprecation notice or are we planning a non-minor release?